### ISSUE TITLE
converted bart model is slower than the original one during inference time

### ISSUE THREAD

**Issue by zangyuchen2008** [NVIDIA/TensorRT#1039]
hi there,
I have a project to use facebook bart for news summerization. In order to make the inference faster, we are trying to convert part of the model to tensorrt and then interegerated into the original model.
Via this repo, I have successfully converted facebook bart decoder layers to tensorrt model, and successfully integerated, however, the total inference time of generated tokens of the new bart model(i.e. the model integerated with converted tensorrt decoder layer) is 2 times slower than the original one, so, I tried to find why, and finally I found that the new bart model itself is faster than the original one, see code below, line1 is faster than before after changing with new bart model,
but is became much slower after line2,

line1: outputs = self(model_inputs, return_dict=True)
line2: next_token_logits = outputs.logits[:, -1, :]
line3: next_token_logits = self.adjust_logits_during_generation(
line4: next_token_logits, cur_len=cur_len, max_length=max_length)

below you can find the comparing speed of new bart model and original one (corresponding to  comparing results of code  line1 above), 
<img width="292" alt="1" src="https://user-images.githubusercontent.com/13851442/106444599-edb8bf80-64b8-11eb-8a55-a76e342b1447.PNG">
below you can find the comparing speed of new bart model and original one(corresponding to  comparing results of code  after line2 above)
<img width="353" alt="2" src="https://user-images.githubusercontent.com/13851442/106444940-615acc80-64b9-11eb-914b-fc6df0f849ee.PNG">
Does anyone knows why it became slow after line1 code above?

**Comment by ttyio** [comment]
Hello @zangyuchen2008 , which precision are you using? have you tried FP16 or INT8.

If you run the onnx using trtexec with option `--verbose --separateProfileRun --dumpProfile`, you can get the layerwise perf breakdown. If you run on both your `line1` model and `line1 + line2 + line3 + line4` model, we might find the reason, thanks!

**Comment by ttyio** [comment]
Closing since no activities for more than 3 weeks, please reopen if you still have question. Thanks!

**Comment by lonelydancer** [comment]
@zangyuchen2008 how's the bart trt model going?
